apps/ui/src/components/Header/index.tsx

```diff
@@ -1,6 +1,10 @@
 import { Col, Row } from 'antd';
 import i18n from 'i18n';
 import React from 'react';
+import { useEffect } from 'react';
+import { useRef } from 'react';
+import { useCallback } from 'react';
+import { useState } from 'react';
```

Comment:
import { a, b, c, d } from 'react'

Reply:
looks like vscode auto import not do it right